builtins/amp-img.js

```diff
@@ -21,7 +21,7 @@ import {guaranteeSrcForSrcsetUnsupportedBrowsers} from '../src/utils/img';
 import {isExperimentOn} from '../src/experiments';
 import {listen} from '../src/event-helper';
 import {registerElement} from '../src/service/custom-element-registry';
-import {setImportantStyles} from '../src/style';
+import {installObjectCropStyles, setImportantStyles} from '../src/style';
```

Comment:
I suggest `propagateObjectFitStyles`

- `propagate` for consistency with `propagateAttributes`
- `ObjectFit` eventhough it is less accurate, it is more clear what it does because it matches one of the CSS properties related to crop.

Reply:
Done